lib/openlogi/api/items.rb

```diff
@@ -0,0 +1,33 @@
+require "openlogi/api/endpoint"
+
+module Openlogi
+  module Api
+    class Items < Endpoint
+      def resource_class
+        Openlogi::Item
+      end
+
+      def find(id, stock: false)
+        params = stock ? { stock: stock } : {}
+        perform_request_with_object(:get, "items/#{id}", params)
+      end
+
+      def all(stock: false)
+        params = stock ? { stock: stock } : {}
+        perform_request_with_objects(:get, "items", params)
+      end
+
+      def update(id, params)
```

Comment:
Actually this doesn't really make sense... there shouldn't be a need to (re-)send the name when updating a record...